apps/connect/src/components/app-loader.tsx

```diff
@@ -1,14 +1,11 @@
-import {
-  App,
-  AppSkeleton,
-  CookieBanner,
-  ModalsContainer,
-  useCheckLatestVersion,
-  useSubscribeToVetraPackages,
-  useSetSentryUser,
-  createReactor,
-} from "@powerhousedao/connect";
-import "../i18n/index.js";
+import { App } from "@powerhousedao/connect/components/app";
+import { AppSkeleton } from "@powerhousedao/connect/components/app-skeleton";
+import { CookieBanner } from "@powerhousedao/connect/components/cookie-banner";
+import { ModalsContainer } from "@powerhousedao/connect/components/modal/modals-container";
+import { useCheckLatestVersion } from "@powerhousedao/connect/hooks";
```

Comment:
When importing from the same "barrel" file, explicit imports must be used. In this case `app-loader` is a component that is importing other components.

`useCheckLatestVersion` can be imported from the hooks barrel file since it won't cause a circular import.
I'm still note sure `vite` is able to tree shake barrel files, so these might have to change to use the full path too. To be tested when all circular imports are fixed.